package.json

```diff
@@ -7,9 +7,7 @@
     "lint": "./node_modules/.bin/eslint --ext .js --ext .jsx packages"
   },
   "devDependencies": {
-    "lerna": "^2.8.0",
```

Comment:
don't remove `babel-eslint` and `es-lint-plugin-react` from the root `package.json` - these are configured at the root of this mono-repo, so the individual packages don't need to do it. Also don't add `babel-eslint` and `lerna` to the package's `package.json`.

Reply:
ok